src/helpers/image-helper.php

```diff
@@ -46,6 +50,13 @@ class Image_Helper {
 	 */
 	private $url_helper;
 
+	/**
+	 * Holds blog charset value for use in DOM parsing.
+	 *
+	 * @var string
+	 */
+	protected $charset = 'UTF-8';
+
```

Comment:
Looks like you're not using this property at all, in which case it's best to get rid of it.

Reply:
Good catch! Forgot to remove it.